### PR TITLE
Install the Service Points feature

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Epic: https://spryker.atlassian.net/browse/CC-26551
Ticket: https://spryker.atlassian.net/browse/CC-29983

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 1]
```diff
@@ -1292,6 +1301,89 @@ Make sure that you can send the following requests:
...
+        return $resourceRelationshipCollection;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that you can send the following requests:
+
+* `GET https://glue.mysprykershop.com/service-points`
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:1381]: * Let's also note the use of "?include=service-point-addresses" for relationship checking.

**Review by abitskil — COMMENTED** [review]
We have to extend the list of modules from the "Make sure that the following modules have been installed:" section.

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 2]
```diff
@@ -1292,6 +1301,89 @@ Make sure that you can send the following requests:
 
 {% endinfo_block %}
 
+3. To enable the Storefront API, register the plugins:
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:1304]:
```suggestion
3. To enable the Storefront API, register the following plugins:
```

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 3]
```diff
@@ -714,13 +721,14 @@ class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
...
+| PaginatedServicePointSearchQueryExpanderPlugin                    | Adds pagination to a search query.                                                                            | None          | Spryker\Client\ServicePointSearch\Plugin\Elasticsearch\Query           |
+| StoreServicePointSearchQueryExpanderPlugin                        | Adds filtering by locale to a search query.                                                                   | None          | Spryker\Client\ServicePointSearch\Plugin\Elasticsearch\Query           |
+| ServiceTypesServicePointSearchQueryExpanderPlugin                 | Adds filtering by service types to a search query.                                                            | None          | Spryker\Client\ServicePointSearch\Plugin\Elasticsearch\Query           |
+| ServicePointAddressRelationExcludeServicePointQueryExpanderPlugin | Excludes service point address relation from query if `excludeAddressRelation` request parameter is provided. | None          | Spryker\Client\ServicePointSearch\Plugin\Elasticsearch\Query           |
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:731]:
```suggestion
| ServicePointAddressRelationExcludeServicePointQueryExpanderPlugin | Excludes service point address relation from query if the `excludeAddressRelation` request parameter is provided. | None          | Spryker\Client\ServicePointSearch\Plugin\Elasticsearch\Query           |
```

**Review by abitskil — APPROVED** [review]

**Review by AlexSlawinski — APPROVED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**